external/fv3fit/fv3fit/keras/_models/shared/__init__.py

```diff
@@ -15,3 +15,5 @@
     standard_normalize,
     count_features,
 )
+from .sequences import XyArraySequence, XyMultiArraySequence, ThreadedSequencePreLoader
+from .halos import append_halos, HalosBatches
```

Comment:
I never ended up using HalosBatches, I think I will remove it. In general it is probably never useful because you don't want output data to have halos.